stable/stolon/templates/proxy-service.yaml

```diff
@@ -23,6 +23,9 @@ spec:
 {{- else }}
   type: {{ .Values.proxy.service.type }}
 {{- end }}
+{{- if .Values.proxy.service.externalIPs }}
```

Comment:
1) Prefer "with" instead of "if" because if any parent segment of the yaml dotpath does not exist it will error.

2) You've added an "if" without a corresponding "end" tag

3) You should add documentation directly in the values.yaml file.